Gemfile

```diff
@@ -6,4 +8,6 @@
 
 source 'https://rubygems.org'
 
+gem 'berkshelf'
```

Comment:
These need to come from ChefDK vs. managing deps in the Gemfile. We used to manage them here and it was a huge maint burden in all the repos

Reply:
Okay, no problem, it helped me to run the tests on my machine but I can revert it.